bin/monitor-manifest

```diff
@@ -1,36 +1,13 @@
 #!/usr/bin/env ruby
 
 require 'listen'
-
-# Write the manifest to allow dynamically loading the images from JS
-# using the correct digest
-def write_manifest
-  File.write(
-    Rails.root / 'app' / 'javascript' / '.config' / 'manifest.json',
-    Propshaft::Assembly.new(Rails.application.config.assets).load_path.manifest
-      .to_json
-  )
-end
-
-# Write a subset of the Exercism config settings to a JSON file
-# to allow using those settings in JS
-def write_env
-  File.write(
-    Rails.root / 'app' / 'javascript' / '.config' / 'env.json',
-    Exercism.config.to_h.slice(:website_assets_host).to_json
-  )
-end
-
-def write_files
-  write_manifest
-  write_env
-end
+require 'generate_js_config'
 
 # Ensure that the files are written when starting
-write_files
+GenerateJSConfig.generate!
 
-unless ENV["EXERCISM_CI"]
-  listener = Listen.to(Rails.root / 'app' / 'images') { write_files }
+if Rails.env.development?
```

Comment:
Lol, good thing I didn't suggest this in https://github.com/exercism/website/pull/2290/commits/4d692740dc5fa4da279deb11ca1dce6a1ab4b271#diff-d4c7ccc36eef0e673446020de7db1e851adc99a30f3a663e8034b05d5e232df8R32 :D